docs/developer/release/README.md

```diff
@@ -0,0 +1,22 @@
+# Release instructions
+
+The documents in this directory are targeted towards core Dangerzone developers
+who want to publish a new Dangerzone release. If you are looking for a way to
+run Dangerzone from source, please check [`BUILD.md`](../../../BUILD.md) instead.
+
+Having a new Dangerzone release out is a rather involved process, because you
+have to do so simultaneously for all the distros that we support. Based on the
+experience from past releases, we have assembled a large list of steps that devs
```

Comment:
```suggestion
experience from past releases, we have assembled a large list of steps to follow
```

Reply:
Fixed in 1859e1cac3f0e987851242ebf939e8ec0f5bdaf5.